tests/Doctrine/Tests/Common/Cache/RedisCacheTest.php

```diff
@@ -35,18 +35,6 @@ public function testGetRedisReturnsInstanceOfRedis()
         $this->assertInstanceOf('Redis', $this->_getCacheDriver()->getRedis());
     }
 
-    public function testSerializerOptionWithOutIgbinaryExtension()
```

Comment:
yes, but unfortunately it's the only way to allow people to choose a serializer different from igbinary when the igbinary module is installed in their environment (which breaks a lot of things on PHP-7 and CommandLine scripts)
If you have a better way to do it I'm all ears. I already thought of adding an extra parameter but that will make things a lot more ugly, and will make the RedisCache even more different from the other implementations.